pyproject.toml

```diff
@@ -4,13 +4,12 @@ build-backend = "hatchling.build"
 
 [project]
 name = "tabpfn-client"
-version = "0.0.14"
+version = "0.0.15"
```

Comment:
Could you update PROD config in google bucket after merging this? :)

Reply:
I will try to do it tomorrow :)